hack/verify-test-owners.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
We have 378 uses of `#!/bin/bash`, and 26 uses of `#!/usr/bin/env bash`. I think assuming bash is /bin/bash is safe.